.github/workflows/release.yml

```
@@ -116,58 +118,31 @@ jobs:
-
       - name: Download dist artifact
         uses: actions/download-artifact@v6
         with:
           name: dist
           path: dist
         continue-on-error: true
 
-      - name: Check if dist contains artifacts
+      - name: Check if dist contains artifacts and not marked as private
         id: check_dist
         run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:11:5: Double quote to prevent globbing and word splitting

(shellcheck)